.circleci/config.yml

```diff
@@ -105,7 +105,7 @@ jobs:
     parallelism: 4
     steps:
       - attach_workspace: *attach_options
-      - run: xvfb-run -a node ./tests/legacy-cli/run_e2e --nb-shards=${CIRCLE_NODE_TOTAL} --shard=${CIRCLE_NODE_INDEX}
+      - run: PATH=~/.npm-global/bin:$PATH xvfb-run -a node ./tests/legacy-cli/run_e2e --nb-shards=${CIRCLE_NODE_TOTAL} --shard=${CIRCLE_NODE_INDEX}
```

Comment:
The `e2e-cli-ng-snapshots` job also needs this change. It only runs on master.

Reply:
thanks